lib/client.js

```diff
@@ -63,22 +67,18 @@ exports.boot = function boot(options, callback) {
       throw new Error('asking to use react router for rendering, but no routes are provided');
     }
 
-    // seems pointless to do this.
-    options.location = options.location || Router.HistoryLocation;
+    // for any component created by react-router, merge model data with the routerProps
+    // NOTE: This may be imposing too large of an opinion?
+    var routerComponent = React.createElement(Router.Router, {
+      createElement: function(Component, routerProps) {
+        return React.createElement(Component, merge({}, props, routerProps));
+      },
 
-    // create and run the router
-    router = Router.create(options);
-
-    router.run(function onRouterRun(Component) {
-
-      // create a component instance
-      var componentInstance = React.createElement(Component, props);
-
-      // finally, render the component instance into the document
-      React.render(componentInstance, mountNode);
+      routes: options.routes,
+      history: routerHistory.createHistory()
     });
-  }
-  else {
+    ReactDOM.render(routerComponent, _document);
```

Comment:
Are we no longer allowing the node to render to via the `options.mountNode`?  If we are no longer using `mountNode` remove it at line 62.